### ISSUE TITLE
Fix TypeError: Cannot read property 'length' of undefined in time-min-milliseconds 

### ISSUE THREAD

**Issue by klimashkin** [stylelint/stylelint#4782]
After upgrading from 13.2.0 to 13.4.0 I've started getting following errors:

1. **Animation**
```css
.progressStop {
  animation: none;
}
```
Throws error
<img width="343" alt="Screen Shot 2020-05-17 at 2 37 27 AM" src="https://user-images.githubusercontent.com/576077/82140946-5bb43e00-97e7-11ea-84b1-56bf155a7abb.png">
Changing it to 
```css
.progressStop {
  animation-name: none;
}
```
fixes the problem

2. **Transition**
This code
```css
.progressTo100Only {
  transition: opacity var(--button-progress-fadeout-duration) var(--button-progress-fadeout-delayTo100Only) ease-out;
}
``` 
make stylelint fail with 
<img width="864" alt="Screen Shot 2020-05-17 at 2 36 30 AM" src="https://user-images.githubusercontent.com/576077/82140927-39babb80-97e7-11ea-8eda-12afe6cdeab1.png">
Could not find a workaround

**Comment by jeddy3** [comment]
@klimashkin Thanks for the report. 

It looks to be a regression. I'll try tracking it down.

Are you turning on the [`time-min-milliseconds`](https://stylelint.io/user-guide/rules/time-min-milliseconds) rule in your config, and are you using it with the [new `ignore: ["delay"]` option](https://stylelint.io/user-guide/rules/time-min-milliseconds#ignore-delay)? Or did the errors occur without any changes to your config after updating to `13.4.0`?

And to confirm, you're updating from `13.2.0` and not `13.3.3`?

Lastly, can you provide a stack trace? Running stylelint on the CLI and outside or gulp should do it, e.g.:

```
npx stylelint "**/*.{css,scss}"
```

**Comment by jeddy3** [comment]
I was able to produce a TypeError using the new `ignore: ["delay"]` option for ` time-min-milliseconds`, but I can't reproduce one otherwise using the information you gave. I'll need more information to go further. In addition to the four questions above, please can you also provide a config that triggers the error?

---

I've created a pull request (https://github.com/stylelint/stylelint/pull/4783) to fix the TypeError for the `"delay`" option

**Comment by klimashkin** [comment]
@jeddy3 Oh shoot, you are right! I forgot that I've actually enabled the new `ignore: ["delay"]` option along with the update, sorry about that.

If I remove it, everything is fine.

**Comment by jeddy3** [comment]
> If I remove it, everything is fine.

I'm glad that's the issue as I was able to reproduce it locally and propose a fix.

**Comment by jeddy3** [comment]
Released in `13.4.1`.
